<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_MainWindow</class>
 <widget class="QMainWindow" name="GUI_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Witcher 3D models converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_baseDir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Base directory :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_folder"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_selectDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="layout_irrlichtRender"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="button_fileSelector">
          <property name="text">
           <string>Select .w2ent file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ImportedFile"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_convert">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Convert -&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_exportedFilename"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_exportedFilename">
          <property name="text">
           <string>Name :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_exportFormat"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_format">
          <property name="text">
           <string>Format :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_logs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Logs :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_log">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_General">
    <property name="title">
     <string>General tools and settings</string>
    </property>
    <addaction name="action_main_Search"/>
    <addaction name="separator"/>
    <addaction name="action_main_Options"/>
    <addaction name="separator"/>
    <addaction name="action_main_Add_mesh"/>
    <addaction name="separator"/>
    <addaction name="action_main_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Languages">
    <property name="title">
     <string>Languages</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="action_display_Wireframe"/>
    <addaction name="action_display_Rigging"/>
    <addaction name="action_display_Normals"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help_About"/>
   </widget>
   <widget class="QMenu" name="menu_Redkit_tools">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Redkit tools</string>
    </property>
    <widget class="QMenu" name="menuLOD">
     <property name="title">
      <string>LOD</string>
     </property>
     <addaction name="action_redkit_LOD0"/>
     <addaction name="action_redkit_LOD1"/>
     <addaction name="action_redkit_LOD2"/>
     <addaction name="separator"/>
     <addaction name="action_redkit_Collision_mesh"/>
     <addaction name="separator"/>
     <addaction name="action_redkit_Clear_current_LOD"/>
     <addaction name="action_redkit_Clear_all_LODs"/>
    </widget>
    <addaction name="action_redkit_Size"/>
    <addaction name="menuLOD"/>
   </widget>
   <widget class="QMenu" name="menu_GameTools">
    <property name="title">
     <string>Game tools</string>
    </property>
    <widget class="QMenu" name="menu_GameTools_The_Witcher_1">
     <property name="title">
      <string>The Witcher 1</string>
     </property>
     <addaction name="action_TW1_BIF_extractor"/>
     <addaction name="separator"/>
     <addaction name="action_TW1_Load_animations"/>
    </widget>
    <widget class="QMenu" name="menu_GameTools_The_Witcher_2">
     <property name="title">
      <string>The Witcher 2</string>
     </property>
     <addaction name="action_TW2_DZIP_extractor"/>
     <addaction name="separator"/>
     <addaction name="action_TW2_Materials_explorer"/>
     <addaction name="action_TW2_Show_linked_files"/>
    </widget>
    <widget class="QMenu" name="menu_GameTools_The_Witcher_3">
     <property name="title">
      <string>The Witcher 3</string>
     </property>
     <addaction name="action_TW3_BUNDLE_extractor"/>
     <addaction name="separator"/>
     <addaction name="action_TW3_Load_rig"/>
     <addaction name="action_TW3_Load_animations"/>
     <addaction name="separator"/>
     <addaction name="action_TW3_Materials_explorer"/>
     <addaction name="action_TW3_Show_linked_files"/>
     <addaction name="action_TW3_LUA_utils_Clean_textures_path_depreciated"/>
    </widget>
    <widget class="QMenu" name="menu_GameTools_TheCouncil">
     <property name="title">
      <string>The Council (probably works with the other Cyanide games)</string>
     </property>
     <addaction name="action_other_TheCouncil_unpacker"/>
     <addaction name="action_other_TheCouncil_Load_template"/>
    </widget>
    <addaction name="menu_GameTools_The_Witcher_1"/>
    <addaction name="menu_GameTools_The_Witcher_2"/>
    <addaction name="menu_GameTools_The_Witcher_3"/>
    <addaction name="menu_GameTools_TheCouncil"/>
    <addaction name="action_other_Dishonored_2_unpacker"/>
   </widget>
   <addaction name="menu_General"/>
   <addaction name="menu_GameTools"/>
   <addaction name="menu_Redkit_tools"/>
   <addaction name="menu_Display"/>
   <addaction name="menu_Languages"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_main_Quitter">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_display_Wireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="action_help_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_main_Search">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_main_Options">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_redkit_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="action_redkit_LOD0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD0 (empty)</string>
   </property>
  </action>
  <action name="action_redkit_LOD1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD1 (empty)</string>
   </property>
  </action>
  <action name="action_redkit_LOD2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD2 (empty)</string>
   </property>
  </action>
  <action name="action_redkit_Clear_current_LOD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear current LOD</string>
   </property>
  </action>
  <action name="action_display_Rigging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="action_redkit_Clear_all_LODs">
   <property name="text">
    <string>Clear all LODs</string>
   </property>
  </action>
  <action name="action_redkit_Collision_mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Convex collision mesh (empty)</string>
   </property>
  </action>
  <action name="action_main_Add_mesh">
   <property name="text">
    <string>Add mesh(es)</string>
   </property>
  </action>
  <action name="action_TW1_BIF_extractor">
   <property name="text">
    <string>BIF extractor</string>
   </property>
  </action>
  <action name="action_TW2_DZIP_extractor">
   <property name="text">
    <string>DZIP Extractor</string>
   </property>
  </action>
  <action name="action_TW2_Materials_explorer">
   <property name="text">
    <string>Materials explorer (TW 2 and 3)</string>
   </property>
  </action>
  <action name="action_TW3_BUNDLE_extractor">
   <property name="text">
    <string>BUNDLE extractor</string>
   </property>
  </action>
  <action name="action_TW3_Load_rig">
   <property name="text">
    <string>Load rig</string>
   </property>
  </action>
  <action name="action_TW3_Load_animations">
   <property name="text">
    <string>Load animations</string>
   </property>
  </action>
  <action name="action_TW3_Materials_explorer">
   <property name="text">
    <string>Materials explorer (TW 2 and 3)</string>
   </property>
  </action>
  <action name="action_TW3_LUA_utils_Clean_textures_path_depreciated">
   <property name="text">
    <string>LUA utils : Clean textures path (depreciated workflow)</string>
   </property>
  </action>
  <action name="action_TW2_Show_linked_files">
   <property name="text">
    <string>Show linked files (TW 2 and 3)</string>
   </property>
  </action>
  <action name="action_TW3_Show_linked_files">
   <property name="text">
    <string>Show linked files (TW 2 and 3)</string>
   </property>
  </action>
  <action name="action_TW1_Load_animations">
   <property name="text">
    <string>Load animations</string>
   </property>
  </action>
  <action name="action_other_Dishonored_2_unpacker">
   <property name="text">
    <string>Dishonored 2 unpacker</string>
   </property>
  </action>
  <action name="action_other_TheCouncil_unpacker">
   <property name="text">
    <string>The Council unpacker</string>
   </property>
  </action>
  <action name="action_other_TheCouncil_Load_template">
   <property name="text">
    <string>Load The Council template</string>
   </property>
  </action>
  <action name="action_display_Normals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normals</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
